vernac/attributes.ml

```diff
@@ -105,16 +121,77 @@ let single_key_parser ~name ~key v prev args =
   assert_once ~name prev;
   v
 
-let bool_attribute ~name ~on ~off : bool option attribute =
-  attribute_of_list [(on, single_key_parser ~name ~key:on true);
-               (off, single_key_parser ~name ~key:off false)]
+(** [key_value_attribute ~name ~default ~keys] parses a attribute [name=key]
+  with possible [key] values in [keys], [default] is for compatibility for users
+  doing [qualif(key)] which is parsed as [qualif(key=default)] *)
```

Comment:
```suggestion
(** [key_value_attribute ~name ~default ~keys] parses an attribute [name=key]
  with possible [key] values in [keys], [default] is for compatibility for users
  doing [qualif(key)] which is parsed as [qualif(key=default)] *)
```

There's also a bit of confusion in this comment introduced by the fact that in `key_value_attribute`, `key` is supposed to be the attribute name (consistent with the last line) but in the first two lines, keys designates the values. I would suggest being consistent and always using `value` to designate what's on the right of `=`. This comment also applies to the named argument in the code below (would make the code more readable IMHO).

Reply:
Good point, will fix!